pkg/dockerregistry/server/manifestservice.go

```diff
@@ -68,7 +68,7 @@ func (m *manifestService) Get(ctx context.Context, dgst digest.Digest, options .
 	ref := imageapi.DockerImageReference{
 		Namespace: m.repo.namespace,
 		Name:      m.repo.name,
-		Registry:  m.repo.config.registryAddr,
+		Registry:  m.repo.app.extraConfig.Server.Addr,
```

Comment:
Can we rename `extraConfig` to `config` ?

Reply:
It's a good idea for follow ups.